v3/services/zip_file_generator.rb

```diff
@@ -0,0 +1,50 @@
+require 'zip'
+
+class ZipFileGenerator
+  # Zip the input directory.
+  def self.write(scr, dst)
+    new(scr, dst).create_zip_file
+  end
+
+  attr_accessor :scr, :dst
+  def initialize(scr, dst)
+    @scr = scr
+    @dst = dst
+  end
+
+  def create_zip_file
```

Comment:
We're already in a class named zip file generator, we don't need to add the extra context of `zip_file` in this method name.